apps/hannk/interpreter/interpreter.h

```diff
@@ -25,10 +25,6 @@ class Interpreter {
     explicit Interpreter(std::unique_ptr<OpGroup> m, InterpreterOptions options = InterpreterOptions());
     ~Interpreter();
 
-    // Return the Tensor in the current Model with the given name.
-    // If none with that name, return null. Tensor is still owned by the Model.
-    TensorPtr get_tensor(const std::string &name);
```

Comment:
This is *definitely* going to be needed at some point. AFAICT it's the API production use cases use to get at the inputs/outputs they want.

Reply:
I'll restore it then.